File: client-node/consumerListagemPessoa.js
var soap = require('soap');
var url = 'http://localhost:8088/server-java/ListagemPessoaService?WSDL';
soap.WSDL.prototype.ignoredNamespaces = [];
var pessoa = {
    'nome':'Chaves',
    'idade':26
}
soap.createClient(url, function(err, client) {
    client.listarPessoa(function(err,result,res){
         console.log('Lista:',result);
    })
    client.validar(pessoa, function(err,result,res){
         console.log('Método validar, sem retorno:',result);
    })
});

File: client-python/consumerListagemPessoa.py
from suds.client import Client
from suds.cache import NoCache
url="http://localhost:8080/server-java/ListagemPessoaService?WSDL"
client = Client(url, cache=NoCache())
print 'Lista: ', client.service.listarPessoa()
#criando o objeto
pessoa = {
    'nome':'Chaves',
    'idade':26
}
client.service.validar(pessoa)
